### ISSUE TITLE
Reopen VS Code debugger not running: Attribute 'program' does not exist #62659

### ISSUE THREAD

**Issue by fluentcodes** [Microsoft/vscode#65621]
I stumbled on the same problem when adding a new folder to my workspace. Everything running beforehand suddenly stops working. Read a lot of solutions on stack overflow and tried some without any effect. 

I had node installed, newest version!

What helped: removing on folder from the workspace. It seems that one can only have one folder in the workspace. 

I'm from IntelliJ and new to code and love your approach. But it seems definitely a bug!

**Comment by isidorn** [comment]
I have no clue what you are talking about, we have 60 thousand issues.
Please give us exact reproducable steps so we can see this on our machine and potentially fix this.

**Comment by fluentcodes** [comment]
Ok. Sorry about that amount of issues... 
I've started with a workspace with node js and everything works fine debugging the code. No big things just some test stuff. 
Then I created a new folder with the real stuff I want to develop. 
I integrated this folder to the existing  workspace and suddenly nothing works any more alway asking to set the launch.json. I looked for solutions for the problem. 
Finally creating a new Workspace and deleting the test stuff everything works fine again. 
Thats all but weired.
